dist/images/ovnkube.sh

```diff
@@ -696,6 +698,56 @@ sb-ovsdb() {
   echo "=============== run sb_ovsdb ========== terminated"
 }
 
+# v3 - Runs ovn-dbchecker on ovnkube-db pod.
+ovn-dbchecker() {
+  trap 'kill $(jobs -p); exit 0' TERM
+  check_ovn_daemonset_version "3"
+  rm -f ${OVN_RUNDIR}/ovn-dbchecker.pid
+  
+  # wait for ready_to_start_node
+  echo "=============== ovn-dbchecker - (wait for ready_to_start_node)"
+  wait_for_event ready_to_start_node
+  echo "ovn_nbdb ${ovn_nbdb}   ovn_sbdb ${ovn_sbdb}"
+
+  # wait for nb-ovsdb and sb-ovsdb to start
+  echo "=============== ovn-dbchecker (wait for nb-ovsdb) ========== OVNKUBE_DB"
+  wait_for_event attempts=15 process_ready ovnnb_db
+
+  echo "=============== ovn-dbchecker (wait for sb-ovsdb) ========== OVNKUBE_DB"
+  wait_for_event attempts=15 process_ready ovnsb_db
+
+  local ovn_db_ssl_opts=""
+  [[ "yes" == ${OVN_SSL_ENABLE} ]] && {
+    ovn_db_ssl_opts="
+        --nb-client-privkey ${ovn_controller_pk}
+        --nb-client-cert ${ovn_controller_cert}
+        --nb-client-cacert ${ovn_ca_cert}
+        --nb-cert-common-name ${ovn_controller_cname}
+        --sb-client-privkey ${ovn_controller_pk}
+        --sb-client-cert ${ovn_controller_cert}
+        --sb-client-cacert ${ovn_ca_cert}
+        --sb-cert-common-name ${ovn_controller_cname}
+      "
+  }
+
+  echo "=============== ovn-dbchecker ========== OVNKUBE_DB"
+  /usr/bin/ovndbchecker \
+    ${K8S_NODE} \
```

Comment:
is this K8S_NODE used at all?

Reply:
No, I had mistakenly not removed that argument to --init-master (copy-paste error). I will remove it.